src/internal/analytics/components/analytics-funnel.tsx

```diff
@@ -51,6 +51,17 @@ export const AnalyticsFunnel = (props: AnalyticsFunnelProps) => {
 
   return <InnerAnalyticsFunnel {...props} />;
 };
+export const CREATION_FLOW_DONE_EVENT_NAME = 'cloudscape-creation-flow-done';
```

Comment:
A general note about the naming: the funnel could also be an edit flow rather than a creation flow. As long as we keep that in mind when listening for the events, I'm fine with this naming.